addons/web/static/src/core/utils/draggable_hook_builder.js

```diff
@@ -500,9 +500,15 @@ export function makeDraggableHook(hookParams) {
                 state.willDrag = false;
 
                 // Compute scrollable parent
-                [ctx.current.scrollParentX, ctx.current.scrollParentY] = getScrollParents(
-                    ctx.current.container
-                );
+                const isDocumentScrollingElement = ctx.current.container
+                    === ctx.current.container.ownerDocument.scrollingElement;
+                // If the container is the "ownerDocument.scrollingElement",
+                // there is no need to get the scroll parent as it is the
+                // scrollable element itself.
+                [ctx.current.scrollParentX, ctx.current.scrollParentY] =
+                    isDocumentScrollingElement
+                    ? [ctx.current.container, ctx.current.container]
+                    : getScrollParents(ctx.current.container);
```

Comment:
Add a "// TODO investigate if `getScrollParents` should not consider the `document.scrollingElement` directly"